packages/core/test/actionCreators.test.ts

```diff
@@ -111,7 +111,10 @@ describe('action creators', () => {
       const resolvedAction = actions.resolveSend(
         action,
         { delay: 100 },
-        toSCXMLEvent({ type: 'EVENT', value: 50 })
+        toSCXMLEvent({ type: 'EVENT', value: 50 } as {
```

Comment:
is this assertion needed? this is not a constrained called anyhow so `toSCXMLEvent` should accept something like `EventObject` or `AnyEventObject` (dont know by heart which one) and `{type:'EVENT',value:50}` should be allowed for this

Reply:
Yes it is. Try removing it - TS doesn't like that `type` is a string instead of `'EVENT'`.